usaspending_api/api_contracts/contracts/award/idv/Download.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Download [/api/v2/download/idvs/]
```

Comment:
DEV-2336 lists the endpoint as `idv` singular, presumably because we're downloading details for a single IDV.